examples/kubernetes/vtgate-service.yaml

```diff
@@ -7,7 +7,10 @@ metadata:
     app: vitess
 spec:
   ports:
-    - port: 15001
+    - name: web+bsonrpc
```

Comment:
I'm not surprised that it choked on the "+".

I'm fine with "web" and changed it accordingly.

vtctld-service.yaml for example also has "web" and "grpc". (But it's also already gRPC only. Therefore, I initially wanted to use the addition "bsonrpc" in the name to reflect the current state.)

And I also agree that the "web" port will go eventually when gRPC supports multiplexing :)